cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly

```diff
@@ -49,6 +48,15 @@ THE SOFTWARE.
                     </l:ajax>
                 </f:dropdownListBlock>
             </j:forEach>
+        </d:tag>
+    </d:taglib>
+    <f:optionalBlock name="snippetizer" title="Snippet Generator" help="/${it.HELP_URL}">
+        <f:section title="Steps"/>
+        <!-- Similar to f:dropdownDescriptorSelector, but adds fallback content to block, and JENKINS-25130 adds per-selection help: -->
+        <f:dropdownList name="prototype" title="Sample Step">
+            <local:listSteps stepDescriptors="${it.getStepDescriptors(false)}"/>
+            <f:dropdownListBlock title="${%— Advanced/Deprecated —}"/>
```

Comment:
Ok, I see what you mean after looking at https://github.com/cloudbees/cloudbees-workflow-docker-plugin/pull/19.

It's probably a NIT because what you have obviously works and is not wrong, but `advanced/deprecated` does not really seem to me like the right descriptor property.